common/common.h

```diff
@@ -370,6 +370,7 @@ struct common_params {
 
     std::string hostname      = "127.0.0.1";
     std::string public_path   = "";                                                                         // NOLINT
+    std::string server_prefix = "";                                                                         // NOLINT
```

Comment:
For consistency, rename the var:

```suggestion
    std::string api_prefix = "";                                                                         // NOLINT
```